libzazen/cuboid_window.c

```diff
@@ -39,7 +43,11 @@ static void zazen_cuboid_window_protocol_request_window_size(
 
   z11_cuboid_window_send_configure(resource, width, height, depth);
 
-  zazen_cuboid_window_update_vertex_buffer(cuboid_window);
+  zazen_cuboid_window_back_state_update(
+      cuboid_window->back_state, cuboid_window->virtual_object,
+      cuboid_window->width, cuboid_window->height, cuboid_window->depth);
+
+  zazen_cuboid_window_update_vertex_buffer(cuboid_window, 1.5);
   zazen_opengl_render_item_set_vertex_buffer(
       cuboid_window->render_item, cuboid_window->vertex_buffer[0],
```

Comment:
```suggestion
      cuboid_window->render_item, cuboid_window->vertex_buffer,
```
今回の変更点じゃないけど、ここコレでいける説ある？

Reply:
そだった、ここは (void *) 型だった。